transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -276,11 +276,11 @@ private void invokeExceptionCaught(final Throwable cause) {
         if (isAdded()) {
             try {
                 handler().exceptionCaught(this, cause);
-            } catch (Throwable t) {
+            } catch (Throwable error) {
                 if (logger.isWarnEnabled()) {
                     logger.warn(
-                            "An exception was thrown by a user handler's " +
-                                    "exceptionCaught() method while handling the following exception:", cause);
+                        "An exception '{}' was thrown by a user handler's exceptionCaught() " +
```

Comment:
Agree. It's just a solution that is slightly better without overhead in terms of huge logs growing as a result of output of two stacktraces in one message. So, if it's fine - could be reworked to full stacktraces.